docs/3.0/develop/transactions.mdx

```diff
@@ -126,7 +126,176 @@ def confirmation(transaction):
 ```
 </Tip>
 
-### Access data within transactions
+## Idempotency
+
+You can ensure sections of code are idempotent by wrapping them in a transaction. By specifying a `key` for your transaction, you can ensure that 
```

Comment:
```suggestion
You can ensure sections of code are functionally idempotent by wrapping them in a transaction. By specifying a `key` for your transaction, you can ensure that 
```

pedantic but i think it's important to distinguish idempotent code from _functional_ idempotency that prevents the code from running twice